tunnel_routes_test.go

```diff
@@ -202,12 +210,13 @@ func TestDeleteTunnelRoute(t *testing.T) {
 			  "tunnel_name": "blog",
 			  "comment": "Example comment for this route",
 			  "created_at": "2021-01-25T18:22:34.317854Z",
-			  "deleted_at": "2021-01-25T18:22:34.317854Z"
+			  "deleted_at": "2021-01-25T18:22:34.317854Z",
+			  "virtual_network_id": "9f322de4-5988-4945-b770-f1d6ac200f86"
             }
           }`)
 	}
 
 	mux.HandleFunc("/accounts/"+testAccountID+"/teamnet/routes/network/10.0.0.0/16", handler)
-	err := client.DeleteTunnelRoute(context.Background(), TunnelRoutesDeleteParams{AccountID: testAccountID, Network: "10.0.0.0/16"})
+	err := client.DeleteTunnelRoute(context.Background(), TunnelRoutesDeleteParams{AccountID: testAccountID, Network: "10.0.0.0/16", VirtualNetworkID: "9f322de4-5988-4945-b770-f1d6ac200f86"})
```

Comment:
we don't need to send `VirtualNetworkID` for the `Delete`, it is only addressable by the `Network` parameter.

Reply:
I have a question here.  For example, 
I have vnetDev and vnetProd. Both of them have "10.0.0.0/16" routes that lead to the corresponding dev & prod tunnels.

How do I remove only vnetProd route without specifying `VirtualNetworkID`?

It's not some specific non-real case. I will have exactly this situation on my environment after we merge this PR.

Delete without `VirtualNetworkID` only removed it for default network.

Reply:
Ah.. I see that it's not reflected in docs. You may test it in Postman/insomnia as I did. I couldn't remove the route from non-default vnet until I explicitly specified vnetid as Get param.

Reply:
gotcha, one to raise with a support ticket if you can in order to get the docs reflecting the correct state of affairs here.

Reply:
Purchased paid cloudflare plan and created a support ticket to change the docs.